install.sh

```diff
@@ -829,8 +829,11 @@ do_unmount_and_remove() {
     set -x
 }
 
+# Cleans the server and agent directory
+rm -rfv /var/lib/rancher/k3s/server/*
+rm -rfv /var/lib/rancher/k3s/agent/*
+
 do_unmount_and_remove '/run/k3s'
-do_unmount_and_remove '/var/lib/rancher/k3s'
```

Comment:
Dumb question: Why don't we just call?
```
do_unmount_and_remove '/var/lib/rancher/k3s/server'
do_unmount_and_remove '/var/lib/rancher/k3s/agent'
```

Reply:
I believe that the main idea from the issue was to not unmount the `server` and the `agent` folders and just clean it up for the user to not have to mount again the `server` and `agent`

[because of this doc](https://docs.k3s.io/reference/resource-profiling#preventing-agents-and-workloads-from-interfering-with-the-cluster-datastore) about mounting the `server` and `agent` to avoid the agent and workload IOPS interfering with the datastore

Reply:
We don't want to unmount the directories themselves, but we still need to unmount and remove everything underneath these paths. Simply running `rm -rf` will not accomplish that.